TheaterSchedule.BLL/DTOs/PerformanceDetailsBaseDTO.cs

```diff
@@ -6,6 +6,7 @@ public class PerformanceDetailsBaseDTO
     {
         public int Duration { get; set; }
         public int MinPrice { get; set; }
+        public int Price { get; set; }
```

Comment:
what about setting minprice = maxprice for case when there is no price range?
And on frontend side just show one value in case when they are equal
What do you think?